### ISSUE TITLE
[Suggestion]: Storage drawers icon rendering range (via drawerfps)

### ISSUE THREAD

**Issue by ivxvm** [DarkPacks/SkyFactory-One#147]
### General Info

- [ ] I would like to propose new content
- [X] I would like to propose changes
- [ ] I would like to propose another thing

### Suggestion Description

There is a mod called drawerfps that prevents item icons on storage drawers from rendering when out of configured range. The default setting is 10, which is too small for anything except rather small closed spaces. I suggest increasing this value to something more reasonable, like 128 (8 chunks) or at least 64.

**Comment by Darkosto** [comment]
Increasing the value to 8 chunks, in my opinion, completely defeats the purpose of this mod. I would say reasonable is a much lower number than that.

**Comment by ivxvm** [comment]
> completely defeats the purpose of this mod
Depends on render distance I guess, if drawers range are equal or more than render distance, then yes, it doesn't ever have an opportunity to hide icons. It seems the issue with their fps is something not affecting everyone equally, because I use a lot of drawers and they don't affect my fps in a noticeable way in two wildly different gaming setups I used. Even changing this value to something smaller than 64 would still be an improvement imo, because the icons disappearing when you move just a few blocks away looks really bad. I would personally prefer to not use them at all if I had to use them with this render range.
